kong/runloop/balancer.lua

```diff
@@ -473,8 +473,13 @@ do
   -- and upstream entity tables as values, or nil+error
   local function load_upstreams_dict_into_memory()
     local upstreams_dict = {}
-    for up in singletons.db.upstreams:each(1000) do
     -- build a dictionary, indexed by the upstream name
+    for up, err in singletons.db.upstreams:each(1000) do
+      if err then
+        log(ngx.CRIT, "could not ensure obtain list of upstreams: ", err)
```

Comment:
This doesn't read very nicely; was a word omitted?

Reply:
oops, it wasn't meant to be in this PR

Reply:
Even better :+1: